### DISCUSSION TITLE
Newer Android will restrict Input Control(Accessibility) settings if the app source is not Play Store / system appstore

### DISCUSSION THREAD

**Discussion by xlionjuan** [rustdesk/rustdesk#9522] [General]
https://support.google.com/android/answer/12623953?hl=en

https://www.reddit.com/r/NothingTech/s/vwWbrhlQI1

Maybe should add a tip or message to let user know how to allow it

![RDT_20240930_1857567142667635570944480.jpg](https://github.com/user-attachments/assets/c386f7f3-e77e-47f2-a9be-2366f048c663)

**Comment by AhsanAliRaza** [comment]
Can somebody guide to me to a resource which show like how to set up the flutter app of it. I want to install this app on a physical app but when i cloned this repo there are a lot of errors in the flutter folder.

**Reply by xlionjuan** [reply]
@AhsanAliRaza Off topic, create new discussion.
